jeeves/blockers.py

```diff
@@ -25,24 +44,15 @@ def get_bugs_dict(bug_ids, config):
 		if bug_id == 0:
```

Comment:
Is a bug_id==0 even possible anymore?

Reply:
This is checking the raw ids from the blocker file I believe, so I think it's still possible in the case where a job has no bugs filed explicitly

Reply:
Unless they are scrubbed somewhere previously?

Reply:
Correct, its not, its addressed with [1].  From a general design approach I would think the method should be responsible for filtering this out (just in case the data passed contained a zero).  But I'm ok with removing it here since [1] effectively handles it.


[1] https://github.com/nathan-weinberg/jeeves/blob/7abde4ebacfc98df54a01e211d4ffb4de6999854/jeeves/report.py#L91

Reply:
Looks like we also have something for this in the set function[1]
I'll need to comb over this more to see if we need all these checks, but for now we should be fine

[1]https://github.com/nathan-weinberg/jeeves/blob/7abde4ebacfc98df54a01e211d4ffb4de6999854/jeeves/blockers.py#L69